Makefile

```diff
@@ -46,6 +46,7 @@ $(CHILD_CONTAINERS):
 push:
 	for i in $(CORE_CONTAINERS) $(CHILD_CONTAINERS); do \
 			[ "$(stagearg)x" = "x" ] && docker push $(registryUrl)/opguk/$$i ; \
+			[ "$(stagearg)x" = "x" ] && docker push $(registryUrl)/opguk/$$i:latest ; \
```

Comment:
Will this get executed in branch builds?

Reply:
yes, but not pushed

Reply:
no, hold on, $(stagearg) means '-dev', so no